src/index.css

```diff
@@ -49,13 +49,6 @@ p {
 	}
 }
 
-:root {
```

Comment:
This might be related to the calendar colour issue mentioned elsewhere

Reply:
Yup! I reverted this change in commit `01e8caf2f53a13cc7628337caf7f36b5537acba0` to resolve it.